src/FormActions.tsx

```diff
@@ -0,0 +1,41 @@
+import { useRecoilValue } from 'recoil'
+import { activeSubmitButtonState, useSelectNodeIndex, useWorkflow } from './store'
+
+export const FormActions = (): JSX.Element => {
+  const index = useSelectNodeIndex()
+  const { deleteNode, clearNodeSelection } = useWorkflow()
+  const submitFormRef = useRecoilValue(activeSubmitButtonState)
+  const { editingGlobal, toggleGlobalEdit } = useWorkflow()
+  if (submitFormRef === undefined || !(index > -1 || editingGlobal)) {
+    return <></>
+  }
+  const DeleteButton = (
+    <button
+      className='btn btn-light'
+      onClick={() => deleteNode(index)}
+      title='Delete node from workflow'
+    >
+      Delete
```

Comment:
small thing: maybe `Delete node` instead?

Reply:
decided to leave as is, because now nodes can be directly deleted from workflow column